app/scripts/controllers/metametrics.js

```diff
@@ -479,13 +479,33 @@ export default class MetaMetricsController {
     }
   }
 
+  // Method to track metrics event
+  // If an actionId is provided createEventFragment, finalizeEventFragment are used to track event,
+  // this is to achieve idempotent behaviour.
+  // If actionid is not provided this._trackEvent is used to track the event.
+  trackEvent(payload, options) {
```

Comment:
Could you format this as a standard JSDoc comment? We should be using JSDoc comments for all public methods.

Reply:
done